### ISSUE TITLE
Addon reports about missing dependencies, then gives error 404 upon clicking to download dependencies

### ISSUE THREAD

**Issue by fatihyuksek1** [muhammadGagah/native-speech-generation#20]
Upon restarting after installing NVDA, the addon mentioned that some dependencies were missing and I clicked ok to download the missing dependencies. And then the addon gave a http 404 error.
Here is the log file:

[nvda.log](https://github.com/user-attachments/files/24589262/nvda.log)

**Comment by fatihyuksek1** [comment]
Hello,
The dependencies where successfully installed after I installed this version.
Thanks for your response.
________________________________
От: muhammadGagah ***@***.***>
Отправлено: 13 января 2026 г. 15:30
Кому: muhammadGagah/native-speech-generation ***@***.***>
Копия: Fatih Yüksek ***@***.***>; Mention ***@***.***>
Тема: Re: [muhammadGagah/native-speech-generation] Addon reports about missing dependencies, then gives error 404 upon clicking to download dependencies (Issue #20)

[https://avatars.githubusercontent.com/u/35881211?s=20&v=4]muhammadGagah left a comment (muhammadGagah/native-speech-generation#20)<https://github.com/muhammadGagah/native-speech-generation/issues/20#issuecomment-3744068961>

Hi @fatihyuksek1<https://github.com/fatihyuksek1> Please try v1.5.2
Here<https://github.com/muhammadGagah/native-speech-generation/releases/tag/v1.5.2>
Has the problem been resolved?

―
